cardano-wallet/src/lib.rs

```diff
@@ -792,6 +792,11 @@ impl SignedTransaction {
     pub fn to_json(&self) -> Result<JsValue, JsValue> {
         JsValue::from_serde(&self.0).map_err(|e| JsValue::from_str(&format! {"{:?}", e}))
     }
+    pub fn from_bytes(bytes: &[u8]) -> SignedTransaction {
```

Comment:
return an error here instead of a pure function `Result<SignedTransaction, JsValue>`